modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -8,12 +8,12 @@
         "event.$": "$",
         "input.$": "$.payload",
         "config": {
-          "buckets.$": "$.meta.buckets",
-          "fileBucketMaps.$": "$.meta.collection.files"
+          "buckets.$": "$.config.buckets",
+          "fileBucketMaps.$": "$.config.fileBucketMaps"
```

Comment:
CMA removes the pathing we'd need for stuff like `.meta.collection.meta` etc...
Opted to just remove that from our definition since we don't care anyway.